docs/quickstart.rst

```diff
@@ -23,6 +23,16 @@ Fairlearn is also available on
 For checking out the latest version in our repository check out our
 :ref:`advanced_install`.
 
+.. note::
+
+    The Fairlearn API is not completely stable, so example code in 
```

Comment:
Two thoughts:
- "not stable" tells me as a user that I don't want to use this. It makes it seem like more of somebody's pet project than what we're doing here. 
- When running an example one should always match versions, that's not caused by Fairlearn being less mature than scikit-learn, for example. If I install a scikit-learn release version and go to their "master" doc page things can break. That's not weird at all, it's expected. That's why their docs point at the last release version. Similarly, I can't expect to install version X and take notebook from version Y and expect that it'll work flawlessly. So the comment is great, but it's not caused by our API being "not completely stable", it's merely making it a bit more frequent.

Reply:
Yes and no..... I take your point about perhaps avoiding the word 'stable' for the reasons you mention. However, I disagree with your statement about always matching versions. I would expect old versions of SciKit-Learn examples to work with newer versions. And the new version of those examples to work with the old SciKit-Learn. If they don't (at least for a couple of versions back), then that's telling me that I'm getting on an update treadmill, and that would not fill me with enthusiasm either.

I'm not suggesting we go for `debian stable`, but we're seeing a lot of users tripped up by this.

Reply:
Thanks for updating. I really like the way it is right now. One could still try and split hairs over whether it's because of the evolving API or whether that's just a factor that makes it worse, but I think that's besides the point and won't get us anywhere. I can very well live with this :-)